dicomImport/matRad_importFieldShapes.m

```diff
@@ -212,8 +223,15 @@
             end
         end
     end
+    openVoxelDistance = [X(shape == 1); Y(shape == 1)];
+    maximumVoxelExtent = max(maximumVoxelExtent, max(abs(openVoxelDistance)));
     tmpCollimation.Fields(i).Shape = shape;
 end
 
+% truncate field shapes to a symmetrical field with limits maximumVoxelExtent
+voxelRange = (-maximumVoxelExtent+1:maximumVoxelExtent) + shapeLimit;
+for i = 1:length(tmpCollimation.Fields) 
```

Comment:
If you crop the opening matrix you also need to adjust tmpCollimation.FieldWidth  - don't you?